grpc.go

```diff
@@ -1791,3 +1798,13 @@ func (a *agentGRPC) StopTracing(ctx context.Context, req *pb.StopTracingRequest)
 
 	return emptyResp, nil
 }
+
+func (a *agentGRPC) GetOOMEvents(ctx context.Context, req *pb.GetOOMEventsRequests) (*pb.OOMEvents, error) {
+	capacity := len(a.sandbox.oomEvents)
```

Comment:
Is it better to wait here until there is an OOM events occurred and then return this call?

Reply:
Yep I can do that if desired. In this case, would we only return a single event at a time?

Reply:
Yeah, I think one event a time can make the api much simple.

Reply:
Updated GRPC endpoint to only return a single event at a time.